src/application.jsx

```diff
@@ -32,7 +33,7 @@ class Application extends React.Component {
               <MemorialPage enJson={this.enJson()}/>
             </Route>
             <Route path='/'>
-              <VeteransPage enJson={this.enJson()}/>
```

Comment:
Do we use the `VeteransPage` component anywhere else, or is the `ChooseRace` component replacing it? If it's replacing it can we delete it?

Reply:
It is still used at the /veterans endpoint; basically Sam didn't want the root page to be VM, he wanted them to select which race they are coming into, so both VM and M10 have their own landing pages after the choose page